### ISSUE TITLE
Add "true" and "false" for checkbox fields integration mapping

### ISSUE THREAD

**Issue by Mosnar** [verbb/formie#819]
**What are you trying to do?**

Salesforce has a weird requirement where all checkbox fields are required to have a value, even if a default is specified in Salesforce. We can work around this currently by mapping to a hidden field.

**What's your proposed solution?**

Add special options for "true" and "false" for boolean checkbox fields within integration mappings - particularly for Salesforce.

**Comment by engram-design** [comment]
Salesforce oddities strike again! Added for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.12",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.5.13](https://github.com/verbb/formie/releases/tag/1.5.13)
